sale_order_product_recommendation_elaboration/wizards/sale_order_recommendation.py

```diff
@@ -38,21 +38,26 @@ class SaleOrderRecommendationLine(models.TransientModel):
     _name = "sale.order.recommendation.line"
     _inherit = ["sale.order.recommendation.line", "product.elaboration.mixin"]
 
-    def _prepare_elaborations_line_form(self, line_form):
+    def _prepare_elaborations_line_form(self, vals):
```

Comment:
Rename method accordingly:

```suggestion
    def _prepare_elaboration_line_vals(self, vals):
```